src/libLookup/Lookup.cpp

```diff
@@ -1601,6 +1599,14 @@ bool Lookup::ProcessSetDSInfoFromSeed(const bytes& message, unsigned int offset,
                     << senderPubKey << " is not in my lookup node list.");
       return false;
     }
+  } else {
+    if ((m_mediator.m_currentEpochNum <= 1) && GUARD_MODE &&
```

Comment:
(m_mediator.m_currentEpochNum <= 1) || (GUARD_MODE && !Guard::GetInstance().IsNodeInDSGuardList(senderPubKey))

Reply:
Implemented but reverted, because we only want to check pubkey for initial epoch